components/infobox/wikis/overwatch/infobox_map_custom.lua

```diff
@@ -99,7 +84,7 @@ function CustomMap:addToLpdb(lpdbData, args)
 	if String.isNotEmpty(args.creator2) then
 		lpdbData.extradata.creator2 = mw.ext.TeamLiquidIntegration.resolve_redirect(args.creator2)
 	end
```

Comment:
```suggestion
	local resolveIfExists = function(value)
		if not value then return end
		return mw.ext.TeamLiquidIntegration.resolve_redirect(value)
	end

	lpdbData.extradata.creator = resolveIfExists(args.creator)
	lpdbData.extradata.creator2 = resolveIfExists(args.creator2)
```

can't span my comment to line above this but that line should be kicked too

mind this is just to avoid trying to redirect resolve nil in case creator should ever be unset